Engine/src/Input/AxisCode.bf

```diff
@@ -0,0 +1,28 @@
+namespace SteelEngine.Input
+{
+	public enum AxisCode
+	{
+		MouseX = 0,
+		MouseY,
+		MouseScroll,
+
+		GamepadLeftStickX,
+		GamepadLeftStickY,
+		GamepadRightStickX,
+		GamepadRightStickY,
+		GamepadLeftTrigger,
+		GamepadRightTrigger,
+
+		MAX
+	}
+
+	public enum GamepadAxisCode : AxisCode
+	{
+		LeftStickX = (.)AxisCode.GamepadLeftStickX,
+		LeftStickY = (.)AxisCode.GamepadLeftStickY,
+		LeftTrigger = (.)AxisCode.GamepadLeftTrigger,
+		RightStickX = (.)AxisCode.GamepadRightStickX,
+		RightStickY = (.)AxisCode.GamepadRightStickY,
+		RightTrigger = (.)AxisCode.GamepadRightTrigger,
```

Comment:
Why is this necessary if we have AxisCode already? Is this just to be more explicit about intent?

Reply:
Okay. Looking later on in the file, I see this is being used for indexing into an array. Can we be explicit about this use case as a comment on this enum?

Reply:
Yeah, that enum is not really necesary. Could add check to the functions that index the array if the value is in range and if not it generates error and returns default.